docs/_pages/en/dsra_scenario_map.md

```diff
@@ -447,40 +447,75 @@ crossorigin=""></script>
   function setBounds() {
 
     if ( lcScenario == "acm7p0_georgiastraitfault" ) {
-      southWest = L.latLng( 48.30891568624434, -128.4312145637652 );
-      northEast = L.latLng( 52.9384673469385, -117.8488971573044 );
+      southWest = L.latLng( 48.30891568684188, -129.0949439967106 );
+      northEast = L.latLng( 53.53110877480622, -117.3589501128889 );
```

Comment:
Just wondering why the extents for these existing scenarios have changed slightly